cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -236,18 +194,5 @@ def configure():
 
 def remove():
     logger.notice('Removing RabbitMQ...')
-    remove_notice(RABBITMQ)
-    remove_logrotate(RABBITMQ)
-    systemd.remove(RABBITMQ)
-    remove_files([
-        HOME_DIR,
-        LOG_DIR,
-        FD_LIMIT_PATH,
-        join('/var/lib', RABBITMQ),
-        join('/usr/lib', RABBITMQ),
-        join('/var/log', RABBITMQ)
-    ])
-    yum_remove('erlang')
```

Comment:
the current convention is that `cfy_manager remove` removes the rpms as well, so we should remove the ones that we install in the `install` phase (all those from `config[RABBITMQ][SOURCES]`)

Reply:
this of course might be subject to change in the future, but in the current transition period let's keep `cfy_manager remove` also uninstalling rpms